plugin-server/src/main/ingestion-queues/session-recording/session-recordings-blob-consumer.ts

```diff
@@ -349,20 +349,19 @@ export class SessionRecordingBlobIngester {
         this.flushInterval = setInterval(() => {
             let sessionManangerBufferSizes = 0
 
+            // in practice, we will always have a values for latestKaftaMessageTimestamp,
+            // but in case we get here before the first message, we use now
+            const kafkaNow = this.latestKafkaMessageTimestamp || DateTime.now().toMillis()
+            const flushThresholdMillis = this.flushThreshold(
+                kafkaNow,
+                DateTime.now().toMillis(),
+                this.serverConfig.SESSION_RECORDING_MAX_BUFFER_AGE_SECONDS * 1000
```

Comment:
Said this before but we should definitely make this an env that we can easily control for the max we want to allow.